spec/libraries/on_page_layout_spec.rb

```diff
@@ -79,6 +79,47 @@ def my_all_callback_method
         end
       end
     end
+
+    context 'when defining two callbacks for different page_layouts' do
+      before do
+        ApplicationController.class_eval do
+          on_page_layout(:standard) do
+            @successful_for_page = true
+          end
+
+          on_page_layout(:news) do
+            @successful_for_page = true
+          end
+        end
+      end
+
+      it 'runs both callbacks' do
+        [:standard, :news].each do |page_layout|
+          alchemy_get :show, urlname: create(:public_page, page_layout: page_layout).urlname
+          expect(assigns(:successful_for_page)).to be_truthy
```

Comment:
Please use `eq(true)` here instead of `be_truthy`, because you set the value to `true` and not any other truthy value. 

Thanks